### PR TITLE
Action API

### PR DESCRIPTION
**Pull request by duckmoll**
#88 
- Changed data model with type enums: 
action_type, entity_id, entity_type, user_account_from, user_account_to, update_time
- Like api with unit tests
- Replace current Account's authentication method with no authentication

### CODE REVIEW

File: new_dawn_server/actions/api/tests/test_like.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            "entity_type": EntityType.MAIN_IMAGE.value,
+            "user_account_from": "/api/v1/account/1/",
+            "user_account_to": "/api/v1/account/2/"
+        }
+
+    def test_like_user(self):
+        res = self.api_client.post(
+            "/api/v1/user_action/", format="json", data=self.like_argument
+        )
+        res_data = json.loads(res.content)
```
Comment (duckmoll) [inline new_dawn_server/actions/api/tests/test_like.py]: Even though I can successfully create Account and UserAction objects, but I get error message from post method: "{'error': "The 'city_preference' field has no data and doesn't allow a null value."}".

Comment (tangziyi001) [reply new_dawn_server/actions/api/tests/test_like.py]: Let me take a look tonight.

Comment (tangziyi001) [reply new_dawn_server/actions/api/tests/test_like.py]: Can you try adding` null = True` here and try again? https://github.com/new-dawn/new_dawn_server/blob/master/new_dawn_server/users/api/resources.py#L198

Comment (duckmoll) [reply new_dawn_server/actions/api/tests/test_like.py]: yes it works now..

**Pull request merged**